app/views/gsa18f/proposals/form.html.erb

```diff
@@ -0,0 +1,75 @@
+<div class="container content">
+  <h3>GSA 18f</h3>
+  <h4>Purchase Request - Supplies</h4>
+  <p>Eg; ergonomic keyboards, foamcore boards, ergonomic mice, monitor stands, post-its, sharpies, etc. </p>
+  <%= form_for @proposal_form, url: @form_url, method: @form_method do |f| %>
+    <!-- Office Location -->
+    <div class="form-group">
+      <%= f.label :office %>
+      <%= f.collection_select(
+            :office, Gsa18f::ProposalForm::OFFICES, :to_s, :to_s,
+            {include_blank: true}, {class: 'form-control'}) %>
+    </div>
+    <!-- Product Name and Description -Text Area- -->
+    <div class="form-group">
+      <%= f.label :product_name_and_description, 'Product Name and Description' %>
+      <%= f.text_area :product_name_and_description, class: 'form-control' %>
+    </div>
+    <!-- Justification -Text Area- -->
+    <div class="form-group">
+      <%= f.label :justification %>
+      <%= f.text_area :justification, class: 'form-control' %>
+    </div>
+    <!-- Link to Product -->
+    <div class="form-group">
+      <%= f.label :link_to_product %>
+      <p> Search GSA Advantage first; then Amazon; then other vendor. Paste link here.</p>
+      <%= f.text_field :link_to_product, class: 'form-control' %>
+    </div>
+    <!-- Amount  -->
+    <div class="form-group">
+      <%= f.label :cost_per_unit %>
+      <div class="input-group">
+        <div class="input-group-addon">$</div>
+        <%= f.number_field :cost_per_unit, class: 'form-control', step: 0.01 %>
+      </div>
+    </div>
+    <!-- Quantity of Product -->
+    <div class="form-group">
+      <%= f.label :quantity %>
+      <div class="input-group">
+        <%= f.number_field :quantity, class: 'form-control', step: 1 %>
+      </div>
+    </div>
+    <!-- Date Requested -->
+    <div class="form-group">
+      <%= f.label :date_requested %>
+      <div class="input-group">
+          <%= f.date_field :date_requested, as: :date, value: f.object.date_requested %>
+      </div>
+    </div>
+    <!-- Urgency -->
+    <div class="form-group">
+      <%= f.label :urgency %>
+      <%= f.collection_select(
+            :urgency, Gsa18f::ProposalForm::URGENCY, :to_s, :to_s,
+            {include_blank: true}, {class: 'form-control'}) %>
+    </div>
+    <!-- Additional Info -->
+    <div class="form-group">
+      <%= f.label :additional_info, 'Please include any additional information you feel we need to correctly execute the purchase' %>
+      <%= f.text_area :additional_info, class: 'form-control' %>
+    </div>
+    <div class="form-group">
+      <%= f.hidden_field :origin, value: 'gsa18f' %>
+    </div>
```

Comment:
Shouldn't be necessary...we should assign the `origin` explicitly in the ProposalForm.